setup.py

```diff
@@ -54,7 +54,7 @@
 print(f"Debug Build: {debug_build}")
 
 if sys.platform == "win32":
-    allocator_libs = []
+    allocator_libs = []  # type: ignore
```

Comment:
Can we remove this? Doesn't seem to be a problem currently. Maybe an outdated pre-commit hook?